pim-tool-front/src/app/component/update-project/update-project.component.ts

```diff
@@ -0,0 +1,107 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Group } from 'src/app/model/group';
+import { Project } from 'src/app/model/project';
+import { GroupService } from 'src/app/service/group.service';
+import { ProjectService } from 'src/app/service/project.service';
+import { SharedService } from 'src/app/service/shared.service';
+
+@Component({
+  selector: 'update-project',
+  templateUrl: './update-project.component.html',
+  styleUrls: ['./update-project.component.scss'],
+})
+export class UpdateProjectComponent implements OnInit {
```

Comment:
[S] Add and Update should share the same Component, they are basically the same.